archivist/proof_mechanism.py

```diff
@@ -11,7 +11,5 @@
 class ProofMechanism(Enum):
     """Enumerate proof mechanism options"""
 
-    #: Assets and events are proven using RKVST Khipus on the ledger
```

Comment:
Would it be prudent to leave some sort of breadcrumb in here for future maintainers to say why we're starting at 2? Is there an accepted pattern in Python for deprecating/reserving enum values?

Reply:
fixed